GamePage.xaml.cs

```diff
@@ -115,10 +115,61 @@ public void ResetDisplay(string word)
 	 */
     private string SelectWord(string gameType)
     {
-		return "placeholder";
+        string[] words;
+        string filePath = "C:\\Users\\alexa\\source\\repos\\wardliii\\Testingexercise\\Resources\\Raw\\wordList.txt";
```

Comment:
this needs to be a relative path to work properly outside of your development environment.